src/test/java/javaslang/control/OptionTest.java

```diff
@@ -131,15 +133,15 @@ public void shouldBePresentOnIsEmptyWhenValueIsPresent() {
 
     @Test
     public void shouldConsumePresentValueOnIsPresentWhenValueIsPresent() {
-        final int[] actual = new int[] { -1 };
+        final int[] actual = new int[]{-1};
```

Comment:
(The benefit of using the same format is, that only _real_ changes are shown in the diff)